src/core/Akka.Remote.Tests/RemoteRouterSpec.cs

```diff
@@ -40,9 +40,9 @@ private class Parent : UntypedActor
         {
             protected override void OnReceive(object message)
             {
-                var tuple = message as Tuple<Props, string>;
-                if (tuple != null)
+                if (message is ValueTuple<Props, string>)
                 {
+                    var tuple = ((Props, string))message;
```

Comment:
this can be removed 
```
  if (message is ValueTuple<Props, string> tuple)
```

Reply:
yes i tried to not force the latest semantics of the language in case you did not wanted too.